src/FxCop/Desktop.Analyzers/Core/Security/DoNotUseInsecureCryptographicAlgorithmsAnalyzer.cs

```diff
@@ -130,30 +153,33 @@ public void AnalyzeNode(SyntaxNodeAnalysisContext context)
 
                 if (type.IsDerivedFrom(this._cryptTypes.DES, baseTypesOnly: true))
                 {
-                    rule = DoNotUseDESSpecificRule;
+                    rule = CreateCA5351DiagnosticDescriptor(owningParentName, _cryptTypes.DES.Name);
```

Comment:
We prefer having a 1:1 mapping between ids and descriptors so that in the ruleset editor, for a given id, there is just one set of thigns to show. In my rewording of the descriptions above, I removed all the placeholders and made it a static string.

Hence also my suggestion to get rid of this method entirely and just create a single descriptor and initialize it to a field.